--- a/columnia/init.lua
+++ b/columnia/init.lua
@@ -665,6 +665,17 @@ columnia.register_column_ia("pinewood",
         "Pineewood Column Linkdown",
         default.node_sound_wood_defaults())
 
+columnia.register_column_ia("acaciawood", "default:acacia_wood",
+        {snappy=2,choppy=2,oddly_breakable_by_hand=2,flammable=3},
+        {"default_acacia_wood.png"},
+        "Acacia Wood Column",
+        "Acacia Wood Column Top",
+        "Acacia Wood Column Bottom",
+        "Acacia Wood Column Crosslink",
+        "Acacia Wood Column Link",
+        "Acacia Wood Column Linkdown",
+        default.node_sound_wood_defaults())
+
 if core.get_modpath( 'moretrees' ) then
 	local morewood = {
 		{ name='beech', description='Beech Tree' },
@@ -675,7 +686,6 @@ if core.get_modpath( 'moretrees' ) then
 		{ name='palm', description='Palm Tree', },
 		{ name='spruce', description='Spruce Tree' },
 		{ name='willow', description='Willow Tree' },
-		{ name='acacia', description='Acacia Tree' },
 		{ name='rubber_tree',	description='Rubber Tree' },
 		{ name='fir', description='Douglas Fir' }
 	}
