--- a/helicopter/heli_control.lua
+++ b/helicopter/heli_control.lua
@@ -157,7 +157,7 @@ function helicopter.heli_control(self, d
         local altitude_consumption_variable = 0
 
         -- if gaining altitude, it consumes more power
-        local y_pos_reference = position.y - 200 --after altitude 200 the power need will increase
+        local y_pos_reference = position.y - 400 --after altitude 200 the power need will increase
         if y_pos_reference > 0 then altitude_consumption_variable = ((y_pos_reference/1000)^2) end
 
         local consumed_power = (power/1500) + altitude_consumption_variable
