--- /dev/null
+++ b/petz/petz/user.conf
@@ -0,0 +1,21 @@
+petz_list = kitty,puppy,ducky,lamb,lion,calf,panda,grizzly,pony,parrot,chicken,piggy,elephant,elephant_female,pigeon,moth,camel,clownfish,bat,silkworm,chimp,hamster,dolphin,tropicalfish,beaver,turtle,frog,toucan,bee,queen_bee,foxy,penguin,butterfly,rat,goat,squirrel,ant,queen_ant,rooster,hen,bunny,wolf,leopard,snow_leopard,warrior_ant
+
+tamagochi_mode = false
+disable_monsters = true
+spawn_chance = 0.05
+max_mobs = 5
+max_per_species = 2
+no_spawn_in_protected = true
+lycanthropy = false
+
+ant_disable_spawn = false
+bee_disable_spawn = false
+chicken_disable_spawn = true
+kitty_disable_spawn = true
+leopard_disable_spawn = true
+lion_disable_spawn = true
+puppy_disable_spawn = true
+rooster_disable_spawn = true
+snow_leopard_disable_spawn = true
+warrior_ant_disable_spawn = true
+wolf_disable_spawn = true
\ No newline at end of file
