--- a/mobs_redo/crafts.lua
+++ b/mobs_redo/crafts.lua
@@ -166,13 +166,13 @@ minetest.register_tool("mobs:shears", {
 	groups = {flammable = 2}
 })
 
-minetest.register_craft({
-	output = "mobs:shears",
-	recipe = {
-		{ "", items.steel_ingot, "" },
-		{ "", items.stick, items.steel_ingot }
-	}
-})
+-- minetest.register_craft({
+-- 	output = "mobs:shears",
+-- 	recipe = {
+-- 		{ "", items.steel_ingot, "" },
+-- 		{ "", items.stick, items.steel_ingot }
+-- 	}
+-- })
 
 -- protection rune
 
