--- a/darkage/init.lua
+++ b/darkage/init.lua
@@ -333,13 +333,13 @@ minetest.register_craft({
 	recipe = "default:desert_stone",
 })
 
-minetest.register_craft({
-	output = "default:desert_stone 2",
-	recipe = {
-		{"default:desert_sand", "default:desert_sand"},
-		{"default:desert_sand", "default:desert_sand"},
-	}
-})
+-- minetest.register_craft({
+-- 	output = "default:desert_stone 2",
+-- 	recipe = {
+-- 		{"default:desert_sand", "default:desert_sand"},
+-- 		{"default:desert_sand", "default:desert_sand"},
+-- 	}
+-- })
 
 -- Tuff
 minetest.register_craft({
