Index: mods/mg_villages/config.lua
===================================================================
--- mods.orig/mg_villages/config.lua
+++ mods/mg_villages/config.lua
@@ -2,11 +2,11 @@
 -- configuration values which you can adjust according to your liking
 -----------------------------------------------------------------------------
 -- set to false if you do not want to have any villages spawning
-mg_villages.ENABLE_VILLAGES = true;
+mg_villages.ENABLE_VILLAGES = false;
 
 -- generate one random building for each mg_villages.INVERSE_HOUSE_DENSITY th mapchunk;
 -- set to 0 in order to disable spawning of these lone buildings outside villages
-mg_villages.INVERSE_HOUSE_DENSITY = 4;
+mg_villages.INVERSE_HOUSE_DENSITY = 0;
 
 -- cover some villages with artificial snow; probability: 1/mg_villages.artificial_snow_probability
 mg_villages.artificial_snow_probability = 10;
@@ -79,10 +79,10 @@ mg_villages.sapling_probability = {};
 
 mg_villages.sapling_probability[ minetest.get_content_id( 'default:sapling' )       ] = 25; -- suitable for a relatively dense forrest of normal trees
 mg_villages.sapling_probability[ minetest.get_content_id( 'default:junglesapling' ) ] = 40; -- jungletrees are a bit bigger and need more space
-mg_villages.sapling_probability[ minetest.get_content_id( 'default:pinesapling' )   ] = 30; 
+mg_villages.sapling_probability[ minetest.get_content_id( 'default:pinesapling' )   ] = 30;
 if( minetest.get_modpath( 'mg' )) then
-	mg_villages.sapling_probability[ minetest.get_content_id( 'mg:savannasapling'     ) ] = 30; 
-	mg_villages.sapling_probability[ minetest.get_content_id( 'mg:pinesapling'        ) ] = 35; 
+	mg_villages.sapling_probability[ minetest.get_content_id( 'mg:savannasapling'     ) ] = 30;
+	mg_villages.sapling_probability[ minetest.get_content_id( 'mg:pinesapling'        ) ] = 35;
 end
 mg_villages.moretrees_treelist = nil;
 if( minetest.get_modpath( 'moretrees' )) then
@@ -119,7 +119,7 @@ end
 -----------------------------------------------------------------------------
 -- how much does the player have to pay for a plot with a building?
 mg_villages.prices = {
-	empty          = "default:copper_ingot 1", -- plot to build on 
+	empty          = "default:copper_ingot 1", -- plot to build on
 
 	-- building types which usually have inhabitants (and thus allow the player
 	-- who bought the building to modifiy the entire village area minus other
