diff --git a/fun_caves/init.lua b/fun_caves/init.lua
index df93ef2..eee2f0d 100644
--- a/fun_caves/init.lua
+++ b/fun_caves/init.lua
@@ -151,9 +151,9 @@ end
 
 minetest.register_on_mapgen_init(function(mgparams)
 	--minetest.set_mapgen_setting('mg_flags', "nocaves, nodungeons", true)
-	minetest.set_mapgen_setting('mg_flags', "nocaves", true)
+	minetest.set_mapgen_setting('mg_flags', "nocaves", false)
 end)
-
-
-dofile(fun_caves_mod.path .. "/mapgen.lua")
+-- 
+-- 
+-- dofile(fun_caves_mod.path .. "/mapgen.lua")
 dofile(fun_caves_mod.path .. "/abms.lua")
